### ISSUE TITLE
Deleting an artist folder, and replacing it with another with the same name and (almost) the same files, you're unable to access to the files again from your library

### ISSUE THREAD

**Issue by Meiden-fc** [airsonic/airsonic#715]
## Problem description

I deleted an artist folder in my server called 'Heredeiros da Crus'. I replaced it with another slightly different that I have in my computer, with some changes on its structure. Then cleanup library, rescan, and I can see the artist name again but with no songs inside.

### Steps to reproduce

1. Delete an existing folder which is part of your library
2. Replace it with another folder with the same name and (almost) same files
3. Re-scan your library
4. Clean-up your library
5. You can access to the artist, but there's no songs that you could play.

## System information

 * **Airsonic version**: 10.1.1-RELEASE Stand-alone war file.
 * **Operating system**: Ubuntu Server 16.0.4.4, 64bit.
 * **Proxy server**: None
 * **Client**: Firefox 58.0.2 on PC and DSub on Android.

## Additional notes
I also tried deleting the whole airsonic directory, and fresh install it, and the problem persists.

I include two screenshots.
![captura de pantalla de 2018-03-11 10-32-47](https://user-images.githubusercontent.com/8806527/37251845-affd3f74-2517-11e8-8968-644bddd3bf6f.png)
![captura de pantalla de 2018-03-11 10-33-28](https://user-images.githubusercontent.com/8806527/37251846-b2f5e168-2517-11e8-9785-80977b780f77.png)

**Comment by Meiden-fc** [comment]
I've followed these steps https://github.com/airsonic/airsonic/issues/650#issuecomment-372479823 to clear-up my library and rebuild it.

1) I stopped airsonic service
2) I deleted /airsonic/thumbs , /airsonic/lastfmcache
3) I deleted airsonic.script lines related to media files (from first INSERT INTO MEDIA_FILE VALUES till the end)
4) Started airsonic server again
5) I added the folders again and my db is exactly identical as before.
6) 'Heredeiros da Crus' is showed as an empty folder again.
